### PR TITLE
Send email

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-2142

### CODE REVIEW

**Review by AbidemiAdio — DISMISSED** [review]
QED

File: changelog/db.changelog-1.0.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet id="1" author="John Watson">
+    <tagDatabase tag="v1.0.0" />
+  </changeSet>
+  <changeSet id="2" author="John Watson">
+    <createTable tableName="statements">
+      <column autoIncrement="true" name="statementId" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="statementId_pkey" />
+      </column>
+      <column name="frn" type="BIGINT" />
```
Comment (marctemp) [inline changelog/db.changelog-1.0.xml:11]: to be consistent with other services we have:

- `name` not `businessName`
- above column is `varchar(127)`
- `addressLine[1,2,3]` then `city` and `county` not `addressLine[4,5]`
- above columns are `varchar(127)`
- `postcode` is `varchar(8)`
- `emailAddress` not `email`
- above column is `varchar(127)`

Comment (johnwatson484) [reply changelog/db.changelog-1.0.xml:11]: The names here match the message format received from constructor -> generator -> publisher.  So feels wrong converting them back to the names specified for DWH here.

Will adjust the sizes to match though.

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**